frontend/integration-tests/protractor.conf.ts

```diff
@@ -16,6 +16,10 @@ import {
 const tap = !!process.env.TAP;
 
 export const BROWSER_TIMEOUT = 15000;
+export const JASMSPEC_TIMEOUT =
+  !_.isNil(process.env.JASMINE_TIMEOUT) && _.isNumber(process.env.JASMINE_TIMEOUT)
+    ? Number(process.env.JASMINE_TIMEOUT)
+    : 120000;
```

Comment:
Don't use `isNil` here since we should treat empty string or 0 like not set. I believe `_.isNumber` will return `false` always for string, so I don't think this will ever be used. Maybe it should just be

```suggestion
export const JASMSPEC_TIMEOUT = process.env.JASMINE_TIMEOUT
    ? Number(process.env.JASMINE_TIMEOUT)
    : 120000;
```

We should doc the new env var in the README.

Reply:
I was hoping the doc/README changes could be delivered in a separate follow-up PR. Is that just not how you do things here?

Reply:
We typically update the README with the PR that makes the change.

Reply:
Okay, we can certainly mention `$JASMINE_TIMEOUT`, as that's clear cut, pretty final. But this PR is not enough to talk about full fledged "Firefox support," as I've not tested debugger experience, I have not tested any other platforms like Mac or Windows. Can we take this in incremental manner (ie. "step1")?

Reply:
Yeah, I'm mainly asking about `$JASMINE_TIMEOUT` since no one will know about it otherwise.

Reply:
Beginner question. How do I "test" my edits in `README.md`? Would I just try it in GUI way on my repo's readme file and copy/paste into this project's?

Reply:
It looks like VSCode will preview markdown if you're using that. Some other suggestions here:

https://stackoverflow.com/questions/9331281/how-can-i-test-what-my-readme-md-file-will-look-like-before-committing-to-github/

Reply:
Ah, thanks, I could have found that info myself. How does this read, right before it starts talking about specific Chrome version in system?
> If you get Jasmine spec timeout errors during runs perhaps against a busy cluster or over slow network, you can try setting a bigger timeout in milliseconds to JASMINE_TIMEOUT environment variable in your shell before running the tests. Default 120000 (2 minutes).

Reply:
Above is in the latest force-push. PTAL.